contracts/src/spaces/facets/membership/MembershipFacet.sol

```diff
@@ -44,6 +44,11 @@ contract MembershipFacet is
     );
   }
 
+  /// @inheritdoc IMembership
+  function revenue() external view returns (uint256) {
```

Comment:
Test?

Reply:
it's a getter

Reply:
can add one later

Reply:
You don't know it's missing in the deploy script if you don't call it.

Reply:
yep, still working on this PR.